include/graphblas/base/vector.hpp

```diff
@@ -236,6 +236,14 @@ namespace grb {
 				(void) n;
 			}
 
+			/**
+			 * Creates a dense ALP/GraphBLAS vector. This constructor takes an initializer
+			 * list of values that will be copied into this vector.
+			 */
```

Comment:
The document should append that this constructor assumes ingesting a dense vector, whose values are given by `vals`. It furthermore should specify that ingestion of `vals` is done assuming sequential I/O semantics.